<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAbout</class>
 <widget class="QDialog" name="DialogAbout">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>32</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>380</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonLogo">
       <property name="minimumSize">
        <size>
         <width>314</width>
         <height>136</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>314</width>
         <height>136</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/new/prefix1/pics/DiEn.bmp</normaloff>:/new/prefix1/pics/DiEn.bmp</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>314</width>
         <height>136</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidgetX" name="tabWidgetAbout">
     <property name="minimumSize">
      <size>
       <width>153</width>
       <height>176</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>176</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabelX" name="labelInfo">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;#VERSION#(#DATE#)&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Copyright (C) 2006-2008 Hellsp@wn&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Copyright (C) 2012-2020 Hors&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Site: &lt;/span&gt;&lt;a href=&quot;http://ntinfo.biz&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;http://ntinfo.biz&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Bugreports: &lt;/span&gt;&lt;a href=&quot;mailto:horsicq@gmail.com&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;horsicq@gmail.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Libraries</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabelX" name="labelComponents">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;QT Library #QT_VERSION# &lt;/span&gt;&lt;a href=&quot;http://qt-project.org&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;http://qt-project.org&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;QWT Library #QWT_VERSION# &lt;/span&gt;&lt;a href=&quot; http://qwt.sourceforge.net&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;http://qwt.sourceforge.net&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabThanks">
      <attribute name="title">
       <string>Thanks</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidgetThanks">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>128</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string notr="true">Nick Sm</string>
          </property>
          <property name="toolTip">
           <string notr="true"/>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">hypn0</string>
          </property>
          <property name="toolTip">
           <string notr="true"/>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">ajax</string>
          </property>
          <property name="toolTip">
           <string notr="true"/>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Hellspawn</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">==DJ==[ZLO]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">ADMIN-CRACK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">mushr00m</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">gurololicon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Diabolic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">SkyRE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">user_</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">void</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">NikolayD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">4kusNick</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">BoRoV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">HandMill</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">ELF_7719116</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">SReg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Zorn</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">BAHEK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">GPcH</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Airenikus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">crc1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">DimitarSerg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">RedBlkJck</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">JMPer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">exet0l</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Mick Grove</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Levis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">sendersu</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">ekse0x</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Slinger</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">DenCoder</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">deniskore</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Dazz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">A.S.L.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">DICI BF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Javier Nieto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Jason Hood</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Tembo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Bartosz Wójcik</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Jeffrey Wu</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Murcielago</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Eliseu Filipi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">dartraiden</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Jupiter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">d3dm4u5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Metabolic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Adler</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">mrexodia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Rinat Aminow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">ktg kbh</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAvatar">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;
border-width:1px;
border-radius:50px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDonate">
      <attribute name="title">
       <string>Donate</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxPayPal">
         <property name="title">
          <string>Paypal</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>ntinfo.re@gmail.com</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxPayPal_2">
         <property name="title">
          <string>Bitcoin</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>3DqddVBX9PKqMvNPXZ3gPHBNNRtD9CnmJo</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>56</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonCheckUpdate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Check update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QTabWidgetX</class>
   <extends>QTabWidget</extends>
   <header>qtabwidgetx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLabelX</class>
   <extends>QLabel</extends>
   <header>qlabelx.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
